src/blog/2024/04/displaying-logged-in-users-on-dashboard.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Displaying logged in user on Node-RED Dashboard 2.0
+subtitle: Step-by-Step Beginner's Guide to Displaying logged in User on Node-RED Dashboard 2.0
+description: Learn how to secure your Dashboard, install, and configure the FlowFuse Multi-user addon, and display logged-in users on Node-RED Dashboard 2.0. Additionally, delve deeper into understanding how the FlowFuse Multi-user addon functions.
+date: 2024-04-02
+authors: ["sumit-shinde"]
+image: 
```

Comment:
@Yndira-E Can you push a tile?

Reply:
@Yndira-E I think we can merge too when you push, can you update the date too?

Reply:
@ZJvandeWeg there were a couple of outstanding points I'd raised that hadn't been resolved:

- https://github.com/FlowFuse/website/pull/1857#discussion_r1547887040
- https://github.com/FlowFuse/website/pull/1857#discussion_r1547872440

Reply:
Hi @joepavitt, I have already updated the intro. Additionally, I have now added a note to emphasize that users must log in to access the Dashboard. Could you please check it?

Reply:
@ZJvandeWeg I just pushed the image